Assets/Scripts/Cards/PistolCards/RapidFire.cs

```diff
@@ -5,19 +5,24 @@
 public class RapidFire : PistolCards
 {
 
-    
+    private int originalLower;
+    private int originalUpper;
+
+
     public override void ExecuteActionEffect()
     {
-        
+
     }
 
     // Start is called before the first frame update
     public override void Initialize()
     {
         lowerBound = 1;
         upperBound = 4;
+        originalLower = lowerBound;
+        originalUpper = upperBound;
         Speed = 2;
-        description = "Attack, Lose 1 accuracy, then make this attack again.";
+        description = "If unstaggered, conditionally make a card consuming accuracy upto 3 times multiplying possible damage."; // () => {accuracy > 3 ? 3 : accuracy} // assume accuracy is deducted
```

Comment:
i'm confused on the wording here; not sure if this retains the original spirit but

If unstaggered, consume X accuracy, up to 3, to make another attack with damage multiplied by X.

Reply:
Good point out 👍

Reply:
I concede I have a propensity to make simple things sound complicated. You're phraseology is correct, would just make explicit if accuracy is 0 nothing happens (it's implied but still) and update damage to lower and upper bounds.